### ISSUE TITLE
Is bi-directional replication supported with pglogical ?

### ISSUE THREAD

**Issue by sunilbhu** [2ndQuadrant/pglogical#315]
Is bi-directional replication works and supported with pglogical for Active-Active ?

Thanks,
Sunil

**Comment by petere** [comment]
pglogical is meant for point-to-point replication. For what you describe, you need something like BDR.

**Comment by sunilbhu** [comment]
Hi Peter, 

Thanks for your reply. I want to use pglogical for Bi-directional replication only ( I have 2 databases , 1 in each data center) Can't I use pglogical for the same ? It would be non-mesh configuration . 

Thank you,
Sunil

**Comment by petere** [comment]
As I said, it is not meant for that. You can try and it might work, but there are complicated issues that you will have to figure out and manage yourself in that case.

**Comment by sunilbhu** [comment]
Thanks Peter. I got your point. 

I tried basic two-way replication with pglogical which was working but I did not explore all use cases those might bring on more challenges . I understand few challenges with pglogical/native logical replication like slot availability after primary failover etc..  Is there any document I can refer which mentions things those BDR can handle but pglogical can't ? That would be very helpful.

Thanks,
Sunil
